enhancements/etcd/automated-backups.md

```diff
@@ -191,9 +239,14 @@ The backup pod needs to be able to save the backup data onto the host filesystem
 
 **TODO:** Does anything prevent us from doing this e.g authorization or security issues.
 
-#### Alerting
+#### Failure to backup and alerting
 
-TBD
+If a backup job fails to complete after a default timeout interval e.g 5 mins the backup controller should set a `BackupControllerDegraded` condition on the `etcds.operator.openshift.io/v1` `cluster` CR. 
+The time to complete a backup should typically be in the range of 30s ( see [upstream FAQ on snapshot timeout](https://etcd.io/docs/v3.5/faq/#what-does-the-etcd-warning-snapshotting-is-taking-more-than-x-seconds-to-finish--mean)). However depending on the size of the data and the hardware this may take longer.
+
+**TODO:** Should we provide an operator flag for overriding the default backup timeout for large or slow clusters?
```

Comment:
maybe add that option to the backup request+schedule CRD? I don't think we want to encourage people to set their CEO to unmanaged in order to configure a timeout